modules/persistent-storage-csi-secrets-store-driver-uninstall.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-secrets-store-driver-uninstall_{context}"]
+= Uninstalling the Secrets Store CSI Driver Operator
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Administrator access to the cluster.
+
+.Procedure
+
+To uninstall the Secrets Store CSI Driver Operator:
+
+. Stop all application pods that use the `secrets-store.csi.k8s.io` provider.
+. Remove the Container Storage Interface (CSI) driver and associated manifests:
+.. Click *Administration* → *CustomResourceDefinitions* → *ClusterCSIDriver*.
+.. On the *Instances* tab, for *secrets-store.csi.k8s.io*, on the far left side, click the drop-down menu, and then click *Delete ClusterCSIDriver*.
+.. When prompted, click *Delete*.
+. Verify that the CSI driver pods are no longer running.
+. Uninstall the Secrets Store CSI Driver Operator:
++
+[NOTE]
+====
+Before you can uninstall the Operator, you must remove the CSI driver first.
```

Comment:
My ignorance: Remove the CSI driver or the CSI diver instance? Does the operator install a CSI driver?